python-sdk/examples/opencv/opencv.py

```diff
@@ -72,12 +72,17 @@ def main():
         # Add some images, recursively inserting content from the images
         # directory. Alternatively, you could use `client.put_file_url` or
         # `client_put_file_bytes`.
-        source = None  # TODO: Add correct path
+        source = relpath("images")
         client.pfs.put_files(commit=commit, source=source, path="/")
```

Comment:
It appears (#9060) that we probably want to use the images from the docs site instead of committing these images to the repo. You should be able to do `client.pfs.put_file_from_url(commit, url="...", path="/")` for each image.